fgrpc/grpcrunner.go

```diff
@@ -41,11 +42,22 @@ const (
 )
 
 // Dial dials grpc either using insecure or using default tls setup.
-// TODO: option to specify certs.
 func Dial(serverAddr string, tls bool) (conn *grpc.ClientConn, err error) {
-	opts := grpc.WithInsecure()
-	if tls || (strings.HasPrefix(serverAddr, "https://")) {
+	var opts grpc.DialOption
+	switch {
+	case tls:
+		tlsCfg, err := ftls.NewCredentials(true, ftls.DefaultClientCert, ftls.DefaultClientKey,
+			ftls.DefaultCACert)
+		if err != nil {
+			log.Errf("Invalid TLS credentials: %v\n", err)
+			return nil, err
+		}
+		creds := credentials.NewTLS(tlsCfg)
+		opts = grpc.WithTransportCredentials(creds)
+	case strings.HasPrefix(serverAddr, "https://"):
```

Comment:
I don't think the switch between standard CA vs custom CA should be based on https:// prefix
it's a config option, to be added to grpcoptions